main.tf

```diff
@@ -144,6 +143,17 @@ resource "aws_elasticache_replication_group" "default" {
   final_snapshot_identifier  = var.final_snapshot_identifier
   apply_immediately          = var.apply_immediately
 
+  dynamic "log_delivery_configuration" {
+    for_each = var.log_delivery_configuration
+
+    content {
+      destination      = lookup(log_delivery_configuration.value, "destination", null)
```

Comment:
are any of these required, or they are all optional?
(if required, we should not provide the default `null` value)

Reply:
The docs do not say. I haven't looked at the golang code and the test sets every key. I figured I'd err on the side of caution and default to null in case one was optional.